r/remarkable/remarkable_ubi_8.3.sh

```diff
@@ -14,18 +16,21 @@
 #             contact "Maintainer" of this script.
 #
 # ----------------------------------------------------------------------------
-yum update -y
-yum -y install git maven java-1.8.0-openjdk.ppc64le java-1.8.0-openjdk-devel.ppc64le
-VERSION=${1:-0.6}
-export JAVA_HOME=/usr/lib/jvm/$(ls /usr/lib/jvm/ | grep -P '^(?=.*java-)(?=.*ppc64le)')
-echo "JAVA_HOME is $JAVA_HOME"
-# update the path env. variable 
-export PATH=$PATH:$JAVA_HOME/bin
+set -e
+PACKAGE_NAME=remarkable
+PACKAGE_VERSION=${1:-v2.0.1}
+PACKAGE_URL=https://github.com/jonschlinkert/remarkable.git
 
+yum install -y git npm
```

Comment:
By default, yum installs node 10.
Please use the node 12 for build and validations.